packages/scaffold/src/partials/w3m-connecting-siwe/index.ts

```diff
@@ -0,0 +1,52 @@
+import { customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { property } from 'lit/decorators.js'
+import styles from './styles.js'
+
+@customElement('w3m-connecting-siwe')
+export class W3mConnectingSiwe extends LitElement {
+  public static override styles = [styles]
+
+  // -- State & Properties -------------------------------- //
+  @property() public dappImageSrc?: string
+
+  @property() public walletImageSrc?: string
+
+  public override firstUpdated() {
+    const visuals = this.shadowRoot?.querySelectorAll('wui-visual-thumbnail')
+
+    if (visuals?.[0]) {
```

Comment:
You could make a private function that takes in a visual and applies these. To deduplicate